main.go

```diff
@@ -92,6 +94,45 @@ func splitChannel(channel string) (string, string, error) {
 	return prefix, version, nil
 }
 
+func doesSliceContain(slice []string, item string) bool {
```

Comment:
there is a new `slices.Contains()` if you want to use it instead

Reply:
perfect, thanks :)